controllers/install/lookout_controller.go

```diff
@@ -145,7 +145,9 @@ type LookoutComponents struct {
 	Service        *corev1.Service
 	ServiceAccount *corev1.ServiceAccount
 	Job            *batchv1.Job
-	// ToDo: add other components
+
+	// ToDo: add other components for Lookout V1
```

Comment:
Would it be difficult to just add this now?  

Either that or create a ticket so you don't forget.